src/components/Quote.js

```diff
@@ -0,0 +1,15 @@
+import React,{useEffect} from "react";
+import { useApp  } from "../context/app-context";
+import { quoteDb } from "../db/quoteDb";
+export const Quote = () => {
+  const { quote, dispatch } = useApp();
+   useEffect(() => {
```

Comment:
Here as well, would like to know if there is any difference in using useEffect because math.random generates a new quote on every refresh

Reply:
same thing I done here